config/config.go

```diff
@@ -79,6 +80,7 @@ func Get() (*Config, error) {
 		ContentTypeByteLimit:                5000000,
 		CookiesControllerURL:                "http://localhost:24100",
 		DatasetControllerURL:                "http://localhost:20200",
+		DatasetFinderEnabled:                true,
```

Comment:
Probably want the default to be set to false; so prod doesn't unintentionally allow users to hit this path

Reply:
Aha, left that in from testing - merci.